pkg/types/args_test.go

```diff
@@ -0,0 +1,113 @@
+package types_test
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+
+	"github.com/appc/cni/pkg/types"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+func TestArgs(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "pkg/ns Suite")
+}
+
+var _ = Describe("UnmarshallableBool UnmarshalText", func() {
+	positive := []string{"True", "true", "1"}
```

Comment:
Could you wrap this and the `negative` before it in a `BeforeEach`. Easier to spot. Variable declarations for both would still be at the top of the `Describe` just before the beforeEach